auto_round/auto_quantizer.py

```diff
@@ -311,7 +312,9 @@ def convert_model(self, model: nn.Module):
         return model
 
     def _dynamic_import_inference_linear(self, bits, backend):
-        if bits == 4 and self.exllama2_available and "exllamav2" in backend:
+        if "qbits" in backend:
+            return qlinear_qbits.QuantLinear
```

Comment:
add itrex in the requirements in someplace to support qbits

Reply:
fix in [ca2aa6e](https://github.com/intel/auto-round/pull/145/commits/ca2aa6efc4cbffb4ea874b8447aca47857c0e935)